composable_prepare_nic_configs.yml

```diff
@@ -327,5 +327,16 @@
               - "{{ isolated_interfaces }}"
               - "{{ openshift_vlan_interfaces }}"
 
-        when: osp_release|int >= 17
+          - name: prepare ceph-storage.j2.j2
+            vars:
+              ctlplane_interface: "{{ ceph_ctlplane_interface }}"
+              external_interface: "{{ ceph_external_interface }}"
+              storage_interface: "{{ ceph_storage_interface }}"
+            template:
+              src: ceph-storage.j2.j2
+              dest: "~/virt/network/vlans/ceph-storage.j2"
+              force: yes
+            delegate_to: undercloud
+            when: ceph_enabled
```

Comment:
I think a `when: osp_release|int < 17 condition` check needs to be added on line 266.

Reply:
yes, i missed it.

Reply:
adding a commit, verify if it looks fine.